Empty databases/openIMIS_ONLINE.sql

```diff
@@ -3613,6 +3623,21 @@ GO
 SET QUOTED_IDENTIFIER ON
 GO
 
+
+CREATE TABLE tblBulkControlNumbers 
+(
+	Id INT IDENTITY(1, 1) CONSTRAINT PK_tblBulkControlNumbers PRIMARY KEY,
+	BillId INT UNIQUE CONSTRAINT UQ_tblBulkControlNumbers_BillId NOT NULL,
+	ProdId INT CONSTRAINT FK_tblBulkControlNumbers_tblProduct FOREIGN KEY REFERENCES tblProduct(ProdId) NOT NULL,
+	OfficerId INT CONSTRAINT FK_tblBulkControlNumbers_tblOfficer FOREIGN KEY REFERENCES tblOfficer(OfficerId) NOT NULL,
+	ControlNumber NVARCHAR(12) NULL,
```

Comment:
this already in TblControlNumber, instead of creating a table why not adding the field in the existing one ?

Reply:
That control number table is different to this. We should not mix these two tables.